src/Platform/Microsoft.Testing.Extensions.VSTestBridge/ObjectModel/RunSettingsAdapter.cs

```diff
@@ -24,6 +24,18 @@ namespace Microsoft.Testing.Extensions.VSTestBridge.ObjectModel;
 /// </summary>
 internal sealed class RunSettingsAdapter : IRunSettings
 {
+    private static readonly string[] UnsupportedRunConfigurationSettings = [
```

Comment:
@Youssef1313 for my ocd, would you mind doing a small follow-up ordering entries by name?